CMakeLists.txt

```diff
@@ -77,14 +84,10 @@ if(BUILD_TESTING)
     enable_testing()
 endif()
 
-# Support RPATH?
-option(ENABLE_RPATH "Enable RPATH for this library" ON)
-mark_as_advanced(ENABLE_RPATH)
-include(AddInstallRPATHSupport)
-add_install_rpath_support(BIN_DIRS "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR}"
-                          LIB_DIRS "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}"
-                          INSTALL_NAME_DIR "${CMAKE_INSTALL_PREFIX}"
-                          DEPENDS ENABLE_RPATH
+# Enable RPATH support for installed binaries and libraries
+add_install_rpath_support(BIN_DIRS "${CMAKE_INSTALL_FULL_LIBDIR}"
+                          LIB_DIRS "${CMAKE_INSTALL_FULL_BINDIR}"
+                          INSTALL_NAME_DIR "${CMAKE_INSTALL_FULL_LIBDIR}"
```

Comment:
Awesome, thanks!
I could not be able to find the `CMAKE_INSTALL_FULL_LIBDIR/BINDIR` is it coming as an utility variables from somewhere? If so, could you please add a comment about it specifying what these variables contain.

Reply:
The `CMAKE_INSTALL_FULL_<dir>` variables are well documented in the [GNUInstallDirs documentation](https://cmake.org/cmake/help/git-master/module/GNUInstallDirs.html) I don't think any explaination here is necessary

Reply:
[My tirivial search criteria got me!](https://cmake.org/cmake/help/latest/search.html?q=CMAKE_INSTALL_FULL_LIBDIR&check_keywords=yes&area=default)
Thanks for pointing to GNUInstallDirs documentation 😄